src/main/java/Edith.java

```diff
@@ -19,10 +20,18 @@ public void talkToUser() {
             if(temporyString.equals("bye")){
```

Comment:
I have noticed no space between if and condition, here and a few other places. You might try look for it in the next commits.